popup.js

```diff
@@ -232,23 +209,33 @@ document.addEventListener('DOMContentLoaded', function() {
     function formatDate(date) {
         return date.toISOString().replace(/[-:]/g, '').split('T')[0];
     }
+
+    function convertToHHMMSS(timeString) {
+        const [time, AMPM] = timeString.split(/(AM|PM)/);
+        let [hours, minutes] = time.split(':');
+
+        if (AMPM === 'PM' && hours < 12) {
+            hours = parseInt(hours, 10) + 12;
+        }
+
+        if (AMPM === 'AM' && hours === 12) {
+            hours = '00';
+        }
+
+        return `${hours}${minutes}00`;
+    }

```

Comment:
prob just use a minified `moment.js` to parse the times -- no point in doing it manually ++ it just adds bloat to the code

Reply:
`timeRanges` should eventually be phased out but currently has no noticeable impact on performance + maintainability cc @drew-747  -- probably just declare this outside the event listener with a constant var, e.g. `TIME_RANGES_MAP`

Reply:
Would it be alright to just include a `moment.min.js` file?

Reply:
should be alr -- should act like a normal js file like `popup.js`